index.js

```diff
@@ -78,7 +78,7 @@ app.on('ready', () => {
       if (fn) fn(win);
 
       // auto updates
-      if (!isDev) {
+      if (!isDev && process.platform !== 'linux') {
```

Comment:
idk, but not need linux [autoupdater](http://electron.atom.io/docs/api/auto-updater/#linux)